### ISSUE TITLE
Integration Tacotron

### ISSUE THREAD

**Issue by twidddj** [twidddj/tf-wavenet_vocoder#2]
So far, I couldn't find the model which attention works in "reduction factor" = 1. If we use the factor > 1, the prediction would seem like below image. It would be a bad news to wavenet performance.
![teacher_forced_mel_prediction](https://user-images.githubusercontent.com/2585210/38550937-cfc4d34c-3cf2-11e8-8064-a80a7c35ec88.png)

Here, the original Mel-spectrum is
![true_mel](https://user-images.githubusercontent.com/2585210/38550957-ddd434a0-3cf2-11e8-8dcd-5951872ba5ba.png)

You can find some discussion for this issue on @Rayhane-mamah's [repo](https://github.com/Rayhane-mamah/Tacotron-2) and @Keithito's [repo](https://github.com/keithito/tacotron/issues/90) also.

**Comment by Rayhane-mamah** [comment]
Hi @twidddj, thanks for sharing your work!

I am assuming you trained the wavenet vocoder on ground truth mels? Did you try training it on ground truth aligned samples generated with the Tacotron ( r > 1 ) model?
In best cases the wavenet will learn to map mels correctly despite the noise in them. If it doesn't work, we'll try figuring out why attention isn't working with r=1. (Have not tested it yet, I get my gpu this week so I'll tell you how it goes)

**Comment by twidddj** [comment]
Hi @Rayhane-mamah, welcome!

Yes, you are right. It has trained on ground truth mels not GTA. I have not tested it yet and have a plan to do it using @Keithito's pretrained model(r=5) in next week. If you tell me how it goes on, it must be very helpful to me. We would get an achievement while we are at it.

**Comment by Rayhane-mamah** [comment]
Yes I am counting on fully training my tacotron an train a wavenet on its
GTA output in the upcoming week, I'll tell you how it goes.

**Comment by twidddj** [comment]
We have tried some works for this issue.
- Tested Rayhane-mamah's Tacotron-2 with r=1. It's attention works and the intelligibility of TTS remarkably improved compared to previous version. However there is [another issue](https://github.com/Rayhane-mamah/Tacotron-2/issues/4#issuecomment-382808698) reported by him. We believe the problem would be solved soon. Thanks!
- Tested our vocoder on the mel spectrograms computed through the same method as Tacotron2 paper( 2048 fft_size, 300 hop_size, 1300 window_size on 24K sample rate). Although it seems to require more training steps(over 1000K) than r9y9's setting, it works too. Thanks to @Ondal90!
